adapters/taboola/taboola.go

```diff
@@ -89,8 +117,50 @@ func (a *adapter) MakeBids(request *openrtb2.BidRequest, requestData *adapters.R
 	return bidResponse, errs
 }
 
-func createTaboolaRequest(request *openrtb2.BidRequest) (taboolaRequest *openrtb2.BidRequest, errors []error) {
+func (a *adapter) buildRequest(request *openrtb2.BidRequest) (*adapters.RequestData, error) {
```

Comment:
`adapter`  need to suffice `adapters.Bidder` interface. Therefore methods like MakeBids,  MakeRequests are defined on `adapter`.    However there is no such need for buildRequest.  So `buildRequest` doesn't need to be defined on `(a *adapter)`

```
func  buildRequest(request *openrtb2.BidRequest) (*adapters.RequestData, error) {
  ...
}
```

Reply:
@onkarvhanumante  It's accessing some struct properties that's why it is defined as a method. 
It can be set as a function and pass the struct properties as arguments.

Reply:
@onkarvhanumante  did the changes